pkg/trimaran/loadvariationriskbalancing/analysis.go

```diff
@@ -73,7 +73,8 @@ func (rs *resourceStats) computeScore(margin float64, sensitivity float64) float
 
 	// evaluate overall risk factor
 	risk := (mu + sigma) / 2
-	klog.V(6).Infof("mu=%f; sigma=%f; margin=%f; sensitivity=%f; risk=%f", mu, sigma, margin, sensitivity, risk)
+	klog.V(6).InfoS("mu=", "mu", mu, " sigma=", "s", sigma, " margin=", "margin", margin, 
```

Comment:
This doesn't look right. Structure logging starts with a general message:

```suggestion
	klog.V(6).InfoS("Evaluating risk factor", "mu", mu, "sigma", sigma, "margin", margin, "sensitivity", sensitivity, "risk", risk)
```